### PR TITLE
Authentication pages

### PR DESCRIPTION
**Pull request by shashidwivedi**
Add Authentication pages
- Sign-up
- Login
- Logout

127 0 0 1_5500_pages_signup html

127 0 0 1_5500_pages_login html

127 0 0 1_5500_pages_logout html

### CODE REVIEW

**Review by srejitk — APPROVED** [review]
Looks good. Please add inline comments for better readability.

File: pages/login.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pillow Willow</title>
+    <link
+      rel="stylesheet"
+      href="https://foobar-ui.netlify.app/styles/foobar-ui.min.css"
+    />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="" />
```
Comment (shubhamkumar648) [inline pages/login.html]: why your crossorigin attributes value is empty

File: pages/signup.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pillow Willow</title>
+    <link
+      rel="stylesheet"
+      href="https://foobar-ui.netlify.app/styles/foobar-ui.min.css"
+    />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="" />
```
Comment (shubhamkumar648) [inline pages/signup.html]: same here crossorigin is empty

**Review by shubhamkumar648 — APPROVED** [review]
consider my comment just a minor changes, the remaining code clean , you can merge

**Review by vaibhav-mahalle — APPROVED** [review]
try to add the relevant files in particular PR. No changes are required.

File: pages/signup.html (hunk 2 above)
Comment (shashidwivedi) [reply pages/signup.html]: Thanks for pointing it out. Fixing

File: pages/login.html (hunk 1 above)
Comment (shashidwivedi) [reply pages/login.html]: done

**Pull request merged**